### PR TITLE
Cleanup cinder liberty conf rebase

### PR DESCRIPTION
**Pull request by dirkmueller**
This was added to the wrong place, moving it to the right place.

### CODE REVIEW

File: chef/cookbooks/cinder/templates/default/cinder.conf.erb [hunk 1]
```diff
@@ -811,6 +806,9 @@ iscsi_write_cache = off
 # content. The cinder backend and locations support must be enabled in the
 # image service, and glance_api_version must be set to 2. (boolean value)
 #image_upload_use_cinder_backend = false
+<% if @show_storage_location -%>
+image_upload_use_cinder_backend = false
```
Comment (aplanas) [inline chef/cookbooks/cinder/templates/default/cinder.conf.erb]: the original logic was to put `true` here

Comment (vuntz) [inline chef/cookbooks/cinder/templates/default/cinder.conf.erb]: s/false/true/ ?

**Comment by mjura** [comment]
+1

**Comment by aplanas** [comment]
+1

**Comment by toabctl** [comment]
+1

**Pull request merged**